lesson_13/homework_13.py

```diff
@@ -16,3 +16,71 @@
     декількох дій, вивести в кого найбільше енергії лишилося.
 Створити тести на клас та на атрибути класу.
 """
+
+
+class Human:
+    def __init__(self, name, second_name, birthday, gender):
+        self.name = name
+        self.second_name = second_name
+        self.birthday = birthday
+        self.gender = gender
+        self.energy = 100
+
+    def eat(self):
+        self.energy += 5
+        if self.energy <= 0:
```

Comment:
```suggestion
        if self.energy <= 0:
```
Перевірка хороша, але коли енергія додається вона не є потрібною, лише коли віднімається